recipes/intarna/build.sh

```diff
@@ -33,7 +33,8 @@ fi
             --with-boost=$PREFIX \
             $CONFIGURE_MULTITHREADING \
             --disable-log-coloring \
-            ${extra_config_options} \
+            --with-boost-libdir=$PREFIX/lib64 \
```

Comment:
$PREFIX/lib should be the path, you can look this up by simply untar the boost tarball.